salt/states/boto_ec2.py

```diff
@@ -169,6 +169,8 @@ def eni_present(
         description=None,
         groups=None,
         source_dest_check=True,
+        allocate_eip=False,
```

Comment:
You know this code far better than I do, so I happily defer to you. Thanks, @ryan-lane :]